src/engine/Settings.cs

```diff
@@ -454,6 +454,9 @@ private Settings()
         Constants.CONTROLLER_DEFAULT_DEADZONE,
     });
 
+    [JsonProperty]
+    public SettingValue<ScreenEffect?> CurrentScreenEffect { get; private set; } = new(null);
```

Comment:
This is a new graphics option, right? So this should be grouped with the other graphics options rather than appended at the end here.